bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/i18n/mybmw.properties

```diff
@@ -1,244 +1,257 @@
-# add-on
-
-addon.mybmw.name = MyBMW
-addon.mybmw.description = Provides access to your Vehicle Data like MyBMW App
+# Binding
+binding.mybmw.name = MyBMW
+binding.mybmw.description = Provides access to your Vehicle Data like MyBMW App
```

Comment:
Please revert this change. You might want to generate the file.

Reply:
OK I don't remember why I changed it. Reverted